rechunker/executors/dask.py

```diff
@@ -6,9 +6,17 @@
 from dask.blockwise import BlockwiseDepDict, blockwise
 from dask.delayed import Delayed
 from dask.highlevelgraph import HighLevelGraph
+from packaging.version import Version
 
 from rechunker.types import ParallelPipelines, Pipeline, PipelineExecutor
 
+# Change in how dask collection token are given to blockwise()
```

Comment:
As of https://github.com/dask/dask/pull/11675, there's a public `dask.task_spec` we can import this from. I think that was released in 2025.1.0.

Could you try adding another version check and importing it from the public location if possible?